flake.nix

```diff
@@ -177,7 +177,7 @@
               darwin.apple_sdk.frameworks.SystemConfiguration
             ])
             ++ lib.optionals (pkgs.stdenv.isLinux) (with pkgs; [
-              checkpolicy
+              # checkpolicy # failed to build 2024-08-08
```

Comment:
Can we find a Nixpkgs that it does build with, or alternatively not update Nixpkgs?